examples/tutorials/logging.ipynb

```diff
@@ -0,0 +1,351 @@
+   "source": [
+    "### Default Log Config\n",
+    "\n",
+    "The log_config.json is the default logging configuration used. \n",
+    "This configuration comes with pre-configured handlers for console level colors, logs, error logs, structured json logs, and fl training logs.\n",
+    "\n",
+    "- The three main sections (formatters, filters, handlers), take care of formatting the structure of the logs, filtering the logs, and sending logs to a destination respectively.\n",
+    "- Under loggers, we configure the root logger and add the handlers\n",
+    "\n",
+    "Let's take a look at the configuration under the server (will be the same for all sites by default):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "befa1e2a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!cat hello-numpy-sag-workspace/server/local/log_config.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ba41cfb",
+   "metadata": {},
+   "source": [
+    "### Default Log Files\n",
+    "\n",
+    "Next we can look at the various log files that are created by the FileHandlers:\n",
+    "\n",
+    "- logFileHandler with baseFormatter to write all logs to log.txt\n",
+    "- jsonFileHandler with jsonFormatter to write json formatted logs to log.json\n",
+    "- errorFileHandler with baseFormatter and level “ERROR” to write error level logs to log_error.txt\n",
+    "- FLFileHandler with baseFormatter and FLFilter to write fl training and custom logs to log_fl.txt\n",
+    "\n",
+    "Feel free to also check out the site logs as well."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5fc69590",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!cat hello-numpy-sag-workspace/server/log.txt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cf576cf0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!cat hello-numpy-sag-workspace/server/log.json"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bb3436da",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!cat hello-numpy-sag-workspace/server/log_error.txt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "526434bf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!cat hello-numpy-sag-workspace/server/log_fl.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d30ee805",
+   "metadata": {},
+   "source": [
+    "### Customization\n",
+    "\n",
+    "The log config file can easily be customized for different use cases.\n",
+    "\n",
+    "For this example, let's assume we are a federated learning researcher mainly interested in the algorithm parts of the log.\n",
+    "By default, we provide the FLFileHandler which uses the FLFilter to generate the log_fl.txt However we would also like to have a cleaner output in the console, as well as remove/shorten some unnecessary log fields.\n",
```

Comment:
suggest adding some reference to nvflare.fuel.utils.log_utils.LoggerNameFilter so we know this FLFilter is actually using that? (I know we have a section, maybe add some texts here to explain?)